assets/js/components/ChecksResults/ChecksSelectionHints.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
I wouldn't create this test. As it is now, this logic will be removed soon and it simply tests the difference in the url.
If we want to check this component (which would be out of scope for this PR), we should test something else

Reply:
Oki i will remove this test